fabfile.py

```diff
@@ -846,7 +846,7 @@ def install_builddeps():
         # may require a sudo
         if not run('brew link libevent', quiet=True):
             sudo('brew link libevent')
-        run('brew install zeromq libtool libmemcached gawk')
+        run('brew install zeromq libtool libmemcached gawk libxmlsec1')
```

Comment:
requirement of saml backend, yes. It's not just speeding up, the XML is cryptographically signed.